run/gem5art/run.py

```diff
@@ -253,6 +246,8 @@ def createFSRun(cls,
         run.hash = run._getHash()
         run.type = 'gem5 run fs'
 
+        # Make the directory if it doesn't exist
+        os.makedirs(run.outdir, exist_ok=True)
```

Comment:
If I am not mistaken, the changes in create(), createFSRun() and createSERun() are for a cleanup and are not necessarily required to enable execution of run in a different directory?

Reply:
Yeah, I didn't a little cleanup. I had to change relative_outdir to outdir, but making outdir a parameter to _create wasn't *required*. This just reduced code duplication.